src/bundles/robot_simulation/engine/Render/helpers/GLTF.ts

```diff
@@ -0,0 +1,31 @@
+import * as THREE from 'three';
+// eslint-disable-next-line import/extensions
+import { type GLTF, GLTFLoader } from 'three/examples/jsm/loaders/GLTFLoader.js';
+import type { Dimension } from '../../Math/Vector';
+
+type GLTFLoaderOptions = Dimension;
+
+export function loadGLTF(url: string, dimension: GLTFLoaderOptions): Promise<GLTF> {
```

Comment:
Interesting that `three` uses `GLTF` capitalisation instead of `Gltf` (thus making it look non-standard), but oh well, I guess we can't do anything about it.